bootstrap/pesticide_seed2.py

```
@@ -0,0 +1,1233 @@
+        return None
+    match = DATE_RE.fullmatch(text)
+    if not match:
+        return None
+    yyyy, mm, dd = match.groups()
+    try:
+        return date(int(yyyy), int(mm), int(dd))
+    except ValueError:
+        return None
+
+
+def parse_int_from_text(value: Any, pattern: re.Pattern[str]) -> int | None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)